scripts/main.py

```diff
@@ -1,13 +1,24 @@
 import argparse
 import trailmap.parse_input as pi
 import trailmap.commodity_dictionary as cd
+import trailmap.acquisition_paths as ap
+# import trailmap.pathway_analysis as pa
+import networkx as nx
+from pprint import pprint
+import matplotlib.pyplot as plt
 
 
 def make_parser():
     """Makes the Cyclus-Trailmap command line parser"""
     p = argparse.ArgumentParser(description="Cyclus-Trailmap command line",
                                 epilog="python main.py cyclus_input_file.xml")
     p.add_argument('infile', nargs=1, help='Cyclus input file. Must be XML')
```

Comment:
you can add a required option to the parameter:
`required - Whether or not the command-line option may be omitted (optionals only).`  [[see](https://docs.python.org/3/library/argparse.html)]